react-ui-kit/src/TimePicker/components/input.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { ChangeEvent, forwardRef, MutableRefObject, RefObject, useState } from 'react';
+
+import { InputWrapper, IconWrapper, StyledInput } from '../StyledComponents';
+import { ReactComponent as CLockOutlineIcon } from '../../Icons/system/ClockOutlineIcon.svg';
```

Comment:
Лучше без Icon, и L маленькая
```suggestion
import { ReactComponent as ClockOutline } from '../../Icons/system/ClockOutline.svg';
```